.buildkite/dagster-buildkite/dagster_buildkite/steps/test_project.py

```diff
@@ -113,6 +117,7 @@ def _test_project_step_key(version: AvailablePythonVersion) -> str:
 
 
 def test_project_depends_fn(version: AvailablePythonVersion, _) -> List[str]:
+    build_for.add(version)
```

Comment:
my understanding was that we build the step just with a skip reason attached, so I would assume this would still get invoked. What am I missing?

Reply:
We're not doing that here - although we probably should for consistency and auditability.

Currently, we're not adding the dependency at all if we have a skip reason:

https://github.com/dagster-io/dagster/blob/46ba805d7ac0258585c65337b3ea2928573a6e09/.buildkite/dagster-buildkite/dagster_buildkite/package_spec.py#L218

Reply:
yike indeed 😞